setup.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
```

Comment:
I don't think these should be directly in the root of the repo

Reply:
This is quite normal for a python package.